SORACOM-LPWAN-with-features.csv

```diff
@@ -132,23 +136,27 @@ Israel,ISR,42502,Cellcom,LTE-M,B3,plan01s,Validated,PSM and eDRX | SMS,01/30/202
 Israel,ISR,42501,Partner,LTE-M,B3,plan01s,Validated,PSM and eDRX | SMS,01/30/2024
 Italy,ITA,22210,Vodafone,LTE-M,B3 | B20,plan01s,Validated,PSM* and eDRX | SMS,01/30/2024
 Italy,ITA,22210,Vodafone,LTE-M,B3 | B20,planX3,Validated,eDRX* (81.92) and PSM* | SMS,06/06/2024
+Japan,JPN,44052,KDDI,LTE-M,B18,plan-KM1,Guaranteed,PSM* and eDRX | SMS,01/12/2024
+Japan,JPN,44051,KDDI,LTE-M,B18,plan01s,Validated,PSM | SMS,07/01/2024
```

Comment:
Could you remove plan01s KDDI (44051) from the list?
We know the network attach itself works but we need to hide it from the political reasons..